### ISSUE TITLE
[Dropdown] A11Y issues 

### ISSUE THREAD

**Issue by JordanWSmith15** [carbon-design-system/carbon-addons-iot-react#1541]
### Describe the bug

The dropdown component has 2 accessibility violations and 1 incomplete
 
### To Reproduce

1. Go to  https://carbon-addons-iot-react.com/?path=/story/dropdown--default
2. Click on 'Accessibility' tab
3. See errors

### Specific timeline issues / requests
MAS 8.3

Blocks https://jsw.ibm.com/browse/GRAPHITE-15954

**Comment by davidicus** [comment]
This looks to be corrected upstream
![image](https://user-images.githubusercontent.com/1590966/107431234-24fc2180-6af4-11eb-802a-7f2ad88b03ef.png)

Closing but feel free to reopen if you are still seeing the problem persist.
